Sources/CardPayments/APIRequests/ConfirmPaymentSourceResponse.swift

```diff
@@ -16,3 +16,4 @@ struct Link: Decodable {
     let href: String?
     let rel, method: String?
 }
+// swiftlint:enable space_after_main_type
```

Comment:
q: Is this because there are multiple types in one file?

Reply:
now that I see it, it is not needed, ill remove it

Reply:
@sshropshire fixed!